documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md

```diff
@@ -0,0 +1,430 @@
+
+### §3.3 Procedure of the evaluation     
+1. The evaluation shall be based on the clearly defined description of the evaluation item in the contract and shall include at least one evaluation of the documentation provided by the applicant (para. 2), inquires and interviews (para. 3) and an on-site evaluation (para. 4). In the case of component certification, technical tests (para. 5) shall be carried out.
+2. With the documentary assessment, the evaluator checks compliance with the requirements of the IDS on the basis of the information in the applicant\'s documentation.
+3. Interviewing the applicant\'s staff or other persons involved in the provisioning of the evaluation item may be used to determine the facts of individual aspects and to verify the accuracy of the documentation.
+In particular, it shall be used to verify aspects identified as critical by the evaluator. Inquiries can be conducted in writing or in person.
+In any case, those concerning central aspects should be conducted as oral interviews.
+Where a personal interview would be disproportionate, it may take the form of videoconferencing.
+4. The on-premise evaluation shall include at least an evaluation of the procedures on the premises of the applicant or its subcontractors.
+5. In the case of component certification, appropriate functional and security tests are conducted.
+6. The evaluation covers all requirements of the IDS for the selected certification level.
+7. The evaluation should be carried out on a sample basis. In relation to a request (e.g. secure passwords), a sample of activities is to be evaluated in order to fulfill a requirement (e.g. password length).
+With regard to the implementation of individual activities, a sample of elements (e.g. individual passwords) shall be evaluated.
+8. The sample shall be selected in such a way that the evaluation of the selected activities or elements provides an indication of the compliance with the requirement.
+
+### §3.4 Recognition of certificates     
+1. The certification body shall recognize relevant certificates for the object of evaluation or individual components thereof.
+The evaluation of the covered criteria for the component named in the certificate is then not necessary.
+However, it is necessary to evaluate criteria not covered by this certificate as well as the interaction of the recognized component with other components, in particular the interfaces relevant to that interaction.
+2. If the applicant seeks recognition, the evaluation facility shall evaluate without delay whether and to what extent recognition can be granted.
+3. The evaluation facility shall ask the certification body for a preliminary ruling on the recognition of certificates. The request shall be accompanied at least by the particulars and documents referred to in § 3.2 (4).
+
+### §3.5 Assessment and evaluation report     
```

Comment:
The term audit / audit report is sometimes used instead of evaluation / evaluation report in this chapter. However, at the moment 3.5 is only about the evaluation report, which should **include** information about all performed (on-site) audits. What the Rules of Procedure do not require at the moment is a separate report for each on-site audit. If this is wanted, it should be explicitly listed as its own requirement in 3.5. Otherwise, with my suggested changes, I would expect all relevant information from the on-site audit to be part of the evaluation report.

Reply:
well spotted. I would agree that so far the report from the on-site audit was planned to the a part of the overall evaluation report and have accepted the changes accordingly